class.jetpack.php

```diff
@@ -436,56 +436,68 @@ class_exists( 'Jetpack_Sitemap_Manager' )
 		delete_transient( self::$plugin_upgrade_lock_key );
 	}
 
-	static function update_active_modules( $modules ) {
-		$current_modules = Jetpack_Options::get_option( 'active_modules', array() );
-
-		$success = Jetpack_Options::update_option( 'active_modules', array_unique( $modules ) );
-
-		if ( is_array( $modules ) && is_array( $current_modules ) ) {
-			$new_active_modules = array_diff( $modules, $current_modules );
-			foreach( $new_active_modules as $module ) {
-				/**
-				 * Fires when a specific module is activated.
-				 *
-				 * @since 1.9.0
-				 *
-				 * @param string $module Module slug.
-				 * @param boolean $success whether the module was activated. @since 4.2
-				 */
-				do_action( 'jetpack_activate_module', $module, $success );
-
-				/**
-				 * Fires when a module is activated.
-				 * The dynamic part of the filter, $module, is the module slug.
-				 *
-				 * @since 1.9.0
-				 *
-				 * @param string $module Module slug.
-				 */
-				do_action( "jetpack_activate_module_$module", $module );
-			}
+	static function isArray( $arr ) {
```

Comment:
It seems we're not using this method anywhere?

Reply:
Thanks for catching. Implemented in [e93c58d](https://github.com/Automattic/jetpack/pull/12270/commits/e93c58dd7aae4af394b22b7ee0fe6a7afc4469e6).